src/app/(admin)/bundles/components/Bundle.tsx

```diff
@@ -137,6 +144,8 @@ export const Bundle = ({ bundleId }: { bundleId: number }) => {
 
         <div>
           <Link href={`/bundles/${bundle.id}/edit`}>Edit</Link>
+          &nbsp;
+          <Link href={{ pathname: generatePublicBundleLink(bundle) }}>Export</Link>
```

Comment:
Only display if we have a valid link.

Reply:
done, I've also added Export Spanish button